variables.tf

```diff
@@ -102,3 +102,8 @@ variable "tfc_url" {
   description = "The Terraform Cloud endpoint.  Must be changed if using Terraform Enterprise."
 }
 
+variable "annotations" {
```

Comment:
I think this might be better as `service_account_annotations` since it only applies there.